### ISSUE TITLE
MAC使用POCO LIB后无法通过开发者验证

### ISSUE THREAD

**Issue by yuan980212** [pocoproject/poco#4040]
如标题所示。

**Comment by bas524** [comment]
Excuse me, but can you provide more information?
Because I don't understand, what does it mean "MAC CANNOT PASS DEVELOPER VERIFICATION AFTER USING POCO LIB".
I believe, you try to write an application, but for what platform?

**Comment by yuan980212** [comment]
就是我的应用在使用POCO LIB之后，MAC的开发者验证虽然通过了，但是在使用其他的设备打开的时候，却提示我无法验证开发者。我在把POCO LIB 去除掉，就可以正常打开。想问下这个会是什么问题？

**Comment by bas524** [comment]
No, I don't know real root cause of problem. But I can suggest you build your application with poco, where poco was builded in static mode.

**Comment by yuan980212** [comment]
我要怎么去编译出带有 openssl 的静态库呢？看文档没有写具体的步骤。

**Comment by bas524** [comment]
You can build poco statically and link openssl dynamically. For simplifying , you can use builds from vcpkg
